apps/ui/src/views/Proposal/Overview.vue

```diff
@@ -32,11 +33,21 @@ const editable = computed(() => {
 });
 
 const cancellable = computed(() => {
-  return (
-    compareAddresses(props.proposal.space.controller, web3.value.account) &&
-    props.proposal.state !== 'executed' &&
-    props.proposal.cancelled === false
-  );
+  if (offchainNetworks.includes(props.proposal.network)) {
+    const addresses = [
+      props.proposal.author.id,
+      props.proposal.space.admins || [],
+      props.proposal.space.moderators || []
+    ].flat();
+
+    return addresses.some(address => compareAddresses(address, web3.value.account));
+  } else {
+    return (
+      compareAddresses(props.proposal.space.controller, web3.value.account) &&
+      props.proposal.state !== 'executed' &&
+      props.proposal.cancelled === false
+    );
+  }
```

Comment:
I'd try to avoid network specific checks like this.

I'm thinking we should:
1. remove `controller` field entirely
2. for SX spaces set `admins` to `[controller]`.
3. update UI so instead of checking for controllers we check for `admins` (or here we will just check first case, without need for `if/else`.
4. in places where we can't use `admins` array (only place I could find was Settings where we edit controller itself we can just use `admins[0]`.

Reply:
How will you handle the admins and moderators ? Some actions on offchain spaces will be admins only, and not available to moderators

Reply:
How those actions are handled on v1? For example if there is something that can be performed by both admins and mods - is admin automatically added to mods list of those are separate and UI checks for both `admins` and `mods`?

I'm thinking about something like this:
1. We unify `controller` to equal to `admin`.
2. We are left out with issue of mods not having replacement on SX spaces, but I assume that all actions that can be done by mods can be done by `controller` - or `admin`. Then we either put controller in `mods` list as well (if admins are included in mods list on v1 already, or leave it empty - if v1 checks for both admins and mods when performing action that can be done by mods).

Would that make sense?

Reply:
admins, mods and controller are 3 different things.

Merging the list is not optimal, as we may use the list to differentiate user's role on the UI.

controller is an entity by itself, and can be neither an admin and mod, and has more permission than an admin.

Reply:
I guess we can keep it like this for now. We can see overtime how the differences affect us in the future (I imagine there can be things that work differently across our networks that we won't/can't unify) and will try to find a solution for it.